Userland/Libraries/LibWeb/CSS/PercentageOr.h

```diff
@@ -88,6 +88,11 @@ class PercentageOr {
         VERIFY_NOT_REACHED();
     }
 
+    virtual T resolve_calculated(NonnullRefPtr<CalculatedStyleValue> const&, [[maybe_unused]] Layout::Node const&, [[maybe_unused]] CSSPixels reference_value) const
```

Comment:
Minor note: You only need `[[maybe_unused]]` on parameters that have names. So you don't need it on the `Layout::Node` here, and could remove it from the `CSSPixels` if you take away its name too. But that's up to you.

Reply:
here I just followed the pattern of `resolve_calculated` defined above for the sake of consistency.

Reply:
No need to be consistent with a mistake :)